counterparty-core/counterpartycore/lib/transaction_helper/p2sh_serializer.py

```diff
@@ -301,3 +314,295 @@ def make_standard_p2sh_multisig_script(multisig_pubkeys, multisig_pubkeys_requir
         bitcoinlib.core.script.OP_CHECKMULTISIG,
     ]
     return multisig_script
+
+
+def select_any_coin_from_source(
+    source,
+    allow_unconfirmed_inputs=True,
+    disable_utxo_locks=False,
+    exclude_utxos=None,
+):
+    """Get the first (biggest) input from the source address"""
+
+    # Array of UTXOs, as retrieved by listunspent function from bitcoind
+    unspent = backend.addrindexrs.get_unspent_txouts(source, unconfirmed=allow_unconfirmed_inputs)
+
+    unspent = UTXOLocks().filter_unspents(source, unspent, exclude_utxos)
+
+    # sort
+    unspent = sort_unspent_txouts(unspent, dust_size=config.DEFAULT_REGULAR_DUST_SIZE)
+
+    # use the first input
+    input = unspent[0]
```

Comment:
Can we fix this warning here? Shouldn't override builtins/primitives

Reply:
fixed